Yotpo/Yotpo/Helper/ApiClient.php

```diff
@@ -89,14 +76,13 @@ public function oauthAuthentication($store_id)
     }
   }
 
-
   public function createApiPost($path, $data, $timeout=self::DEFAULT_TIMEOUT) {
     try 
     {
-      $config = array('timeout' => $timeout);
+      $cfg = array('timeout' => $timeout);
```

Comment:
cfg is not that longer then config.... please leave it as config the code is more readable...